app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+    render json: @users
```

Comment:
While leveraging the built-in `to_json` is definitely the simplest thing that works, right now, what you are essentially doing is to the structure of the user as it is stored in the database.

Leveraging the built-in `to_json` has a couple of repercussions:
1. Whenever you add a column to the database, it will be returned via the API. This can be good, but it can also be unexpected. In a situation where you have multiple developers on the same team, one developer may end up exposing data without even realizing it.
2. Altering the structure of your schema will cause changes to the API.
3. One has to look at the ActiveRecord model and/or database schema in order to know what is being exposed from a given endpoint.

Can you think of a way to refactor this code to address some of the issues listed above?